sdc/utilities/prange_utils.py

```diff
@@ -54,36 +55,19 @@ def get_pool_size_impl():
     return get_pool_size_impl
 
 
+@sdc_register_jitable
 def get_chunks(size, pool_size=0):
     if pool_size == 0:
         pool_size = get_pool_size()
 
-    chunk_size = (size - 1) // pool_size + 1
+    chunk_size = size // pool_size
```

Comment:
What's not valid.

What are you doing?

Reply:
After closer look, it is valid, but a bad idea.

Let's assume, you have 56 threads and task size 55. In your implementation it wouldn't be paralleled at all. All 55 would be computed by the last chunk

Reply:
It was my idea :) After some thinking I have the same conclusion. We should do the last thread to take lesser work
.

Reply:
I have implemented my vision in #618.